packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -817,4 +817,80 @@ void main() {
     final TextField textFieldWidget = tester.widget(textFieldFinder);
     expect(textFieldWidget.scrollController, scrollController);
   });
+  
+  testWidgets('TextFormField changes mouse cursor when hovered', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Material(
+          child: MouseRegion(
+            cursor: SystemMouseCursors.forbidden,
+            child: TextFormField(
+              mouseCursor: SystemMouseCursors.grab,
+              decoration: InputDecoration(
+                // Add an icon so that the left edge is not the text area
+                icon: Icon(Icons.person),
+              ),
```

Comment:
Do the tests still work if we remove this (ditto to the second case), since this is used for `TextArea`?

Reply:
Seems the test works fine without the `decoration`, so removed in both of the test cases.